subprojects/kotlin-dsl/src/main/kotlin/org/gradle/kotlin/dsl/support/delegates/ProjectDelegate.kt

```diff
@@ -154,6 +154,9 @@ abstract class ProjectDelegate : Project {
     override fun zipTree(zipPath: Any): FileTree =
         delegate.zipTree(zipPath)
 
+    override fun zipTree(args: MutableMap<String, *>): FileTree =
```

Comment:
```suggestion
    override fun zipTree(args: Map<String, *>): FileTree =
```

Why is it mutable?

Reply:
I think I used IDEA's auto-creation that set this up. I don't see a reason to support MutableMaps.